lib/js/components/DatePickers/DateRangePicker/DateRangePicker.vue

```diff
@@ -158,6 +158,11 @@ export default defineComponent({
 			emit('update:date', { startDate: event[0], endDate: event[1] });
 		};
 
+		const onClose = () => {
+			destroyDatePicker();
```

Comment:
You don't have to import this method. You can call `destroyDatePicker` in composable. The only thing would be this ref cleanup which is good enough for me